docs/UserGuide.md

```diff
@@ -47,140 +44,200 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the task list.<br>
+Format: `add t/TASK [d/DATE] [#tags]`
 
 > Persons can have any number of tags (including 0)
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add t/CS2103 meeting d/03/03 #Programming Lab 2`
 
-### 2.3. Listing all persons : `list`
+### 2.3. Listing all tasks : `list`
 
-Shows a list of all persons in the address book.<br>
+Shows a list of all tasks in the task list.<br>
 Format: `list`
 
-### 2.4. Editing a person : `edit`
+### 2.4. Create a list: `createlist`
+
+Create a new list. <br>
```

Comment:
what does this mean? need to explain what it is, else how will the user know what to do with this command?